### PR TITLE
flare-game: update to 1.14

### PR DESCRIPTION
**Pull request by MIvanchev**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by ahesford** [comment]
You are welcome to update these packages, but please defer adoption until you have a longer history of contributions to Void.

**Comment by MIvanchev** [comment]
> You are welcome to update these packages, but please defer adoption until you have a longer history of contributions to Void.

Sorry, I missed this message @ahesford. I mean how much longer can my history get :) I'll remove the adoption if you insist.

**Comment by MIvanchev** [comment]
Bump.

**Comment by ahesford** [comment]
You never undid the adoption.

**Comment by MIvanchev** [comment]
Oh, didn't get that you indeed insisted on the removal, just a moment.

**Comment by ahesford** [comment]
Well I didn't ask you to do it as a joke...

**Comment by MIvanchev** [comment]
Well not as a joke but more of a discussion 🤭  I did plan to take care of it long term.

File: srcpkgs/flare-engine/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-version=1.13.04
+version=1.14
 revision=1
 build_style=cmake
 configure_args="-DBINDIR=/usr/bin -DDATADIR=share/flare"
 makedepends="SDL2_image-devel SDL2_mixer-devel SDL2_ttf-devel"
 depends="desktop-file-utils hicolor-icon-theme"
 short_desc="Free/libre action roleplaying engine (engine only)"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Mihail Ivanchev <contact@ivanchev.net>"
```
Comment (ahesford) [inline srcpkgs/flare-engine/template:10]: Don't forget to revert this line too.

Comment (MIvanchev) [reply srcpkgs/flare-engine/template:10]: Damn I forgot that

**Pull request merged**